### PR TITLE
Add matta

### LINKED ISSUE #42: 待った に対応する
あえて 待った をありにしたい。

### PR DESCRIPTION
**Pull request by kouki-dan**
Close #42

### CODE REVIEW

**Comment by kouki-dan** [comment]
Review me

File: app/shogi.py [hunk 1]
```diff
@@ -113,3 +113,19 @@ def resign(channel, message):
     message.send(board_str)
     ShogiInput.clear(channel.channel_id)
 
+@respond_to("待った")
```
Comment (millay1114) [inline app/shogi.py]: please add ``まった`` (dogeza-emoji)(dogeza-emoji)(dogeza-emoji)

Comment (kouki-dan) [inline app/shogi.py]: No way!

Comment (millay1114) [inline app/shogi.py]: (shobo~n-emoji).

Comment (millay1114) [inline app/shogi.py]: :-1:

**Comment by millay1114** [comment]
:+1: +1:

**Pull request merged**